tests/test_kubeseal.py

```
@@ -215,14 +225,40 @@ def test_create_regcred_secret(self, kubeseal_mocks, mock_subprocess):  # noqa:
+        """Test creating a generic secret with non-existent file raises ClickException."""
+        kubeseal = Kubeseal(select_context=False)
+        secret_params = SecretParams(name="test-secret", namespace="default", secret_type=SecretType.GENERIC)
+
+        with (
+            patch("questionary.select") as mock_select,
+            patch("questionary.path") as mock_path,
+            pytest.raises(click.ClickException) as exc_info,
+        ):
+            mock_select.return_value.unsafe_ask.side_effect = ["file", "done"]
+            mock_path.return_value.unsafe_ask.return_value = "nonexistent.json"
+            kubeseal.create_generic_secret(secret_params)
```

Ruff (0.14.7)

`pytest.raises()` block should contain a single simple statement

(PT012)